views/importing/queue_items_grouped_album.html

```
@@ -125,6 +125,50 @@ <h4 class="text-sm font-medium text-gray-900 dark:text-white">{{.Track.Title}}</
                               {{end}}
                         </div>
                     </div>
+                    <!-- Audio players -->
+                    {{if eq .Type "duplicate"}}
+                    <div class="flex flex-wrap items-center gap-1.5">
+                        <button _="on click toggle .hidden on #pa-queue-{{.ID}}"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)